task-definition.plugins.json

```diff
@@ -161,6 +161,13 @@
             ],
             "linuxParameters": {
                 "initProcessEnabled": true
+            },
+            "healthCheck": {
+                "retries": 10,
+                "command": ["CMD-SHELL", "curl -f http://localhost:8000/_ready || exit 1"],
```

Comment:
why isn't this _health?

Reply:
Because ECS doesn't have the concept of a readiness check like k8s does. 

And while we can expand on this further, for now I'd just like a check that returns: "plugin server ready to ingest events" so we don't flip to the new tasks until that passes, preventing backpressure and the like on deploys